src/components/Touchable.jsx

```diff
@@ -60,19 +63,42 @@ const StyledRipple = styled(Ripple)`
  *
  */
 
+const defaultRippleConfig = {
+  rippleCentered: false,
+  rippleSequential: false,
+  rippleFades: true,
+  rippleOutsideContainer: false,
+  rippleColor: "#000",
+  rippleOpacity: 0.09,
+  rippleDuration: 600,
+  rippleSize: 0,
+  rippleContainerBorderRadius: 0,
+};
+
 export const Touchable = React.forwardRef((props, ref) => {
-  const { children, elevation, ...rest } = props;
+  const { children, elevation, rippleConfig, ...rest } = props;
   const shadowStyles = elevation ? getShadowStyles(elevation) : {};
 
+  const rippleProps = {
+    ...defaultRippleConfig,
+    rippleCentered:
+      rippleConfig?.isCentered ?? defaultRippleConfig.rippleCentered,
+    rippleSequential:
+      rippleConfig?.isSequential ?? defaultRippleConfig.rippleSequential,
+    rippleFades: rippleConfig?.shouldFade ?? defaultRippleConfig.rippleFades,
+    rippleOutsideContainer:
+      rippleConfig?.shouldOverflowContainer ??
+      defaultRippleConfig.rippleOutsideContainer,
+    rippleColor: rippleConfig?.color ?? defaultRippleConfig.rippleColor,
+    rippleOpacity: rippleConfig?.opacity ?? defaultRippleConfig.rippleOpacity,
+    rippleDuration:
+      rippleConfig?.duration ?? defaultRippleConfig.rippleDuration,
+    rippleSize: rippleConfig?.size ?? defaultRippleConfig.rippleSize,
```

Comment:
@chiragbhaiji  Please check and merge.

https://app.usebubbles.com/81MEskQn71Bpuoui6zYdgf/untitled-bubble

Reply:
Done.